oak/include/oak/server.hpp

```diff
@@ -123,6 +106,10 @@ class Server {
    */
   void set_threads(int n) { listen_threads_ = n; }
 
+  /**
+   * Get the address on which the server listens
+   */
+  inline const auto& address() const { return listen_addr_; }
```

Comment:
Inside class definitions, `inline` is implicit.
See: https://isocpp.org/wiki/faq/inline-functions#inline-member-fns-more

Also, remember that `inline` is just a compiler hint.

Ditto for the rest of the file.